packages/notifier/src/main.ts

```diff
@@ -1,8 +1,28 @@
 import * as core from '@actions/core'
+import * as gh from '@actions/github'
```

Comment:
I would probably have done:


```suggestion
import { whatever, is, needed } from '@actions/github'
```

Reply:
Is there a downside to importing `*`? This is how many first party actions do it, too.

Reply:
Unsure if tree shaking works if you import everything, but probably not.

It's just not as idiomatic in an es6 world. With es5 this was normal because you'd only have one export. So you'd import a namespace. 

Now it's converting all the exports into a namespace and the using them. In my.opinion it's the equivalent of type juggling.